samples/RewriteSample/Startup.cs

```diff
@@ -11,14 +11,14 @@ namespace RewriteSample
 {
     public class Startup
     {
-        public void Configure(IApplicationBuilder app, IHostingEnvironment hostingEnvironment)
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
```

Comment:
We would probably need to add a new `RewriteOptionsBuilder` API in that case. I'm not sure semantics of `.SetBasePath` on  `RewriteOptions` are clear